server/lib/tuist_web/marketing/controllers/marketing_controller.ex

```diff
@@ -448,6 +449,26 @@ defmodule TuistWeb.Marketing.MarketingController do
     |> render(:blog_atom, layout: false)
   end
 
+  def case_studies_rss(conn, _params) do
+    cases = CaseStudies.get_cases()
+    last_build_date = cases |> List.first() |> Map.get(:date)
+
+    conn
+    |> assign(:cases, cases)
+    |> assign(:last_build_date, last_build_date)
+    |> render(:case_studies_rss, layout: false)
+  end
+
+  def case_studies_atom(conn, _params) do
+    cases = CaseStudies.get_cases()
+    last_build_date = cases |> List.first() |> Map.get(:date)
+
+    conn
+    |> assign(:cases, cases)
+    |> assign(:last_build_date, last_build_date)
+    |> render(:case_studies_atom, layout: false)
+  end
```

Comment:
I don't know if we want to have _another_ RSS for case studies. Case studies are basically special pieces of blog posts and I think we should try to model them more like that.

Reply:
> Is it that they should show up in the RSS & Atom of the blog?

That and I would expect them to show up in the blog page as well: https://tuist.dev/blog